F.A.Q testing

alexevic opened the pull request:
I created full F.A.Q. page TA. Fixed some naming in vision TA.

alexevic committed:
faq.feature file was created.

Manual Gherkin tests for F.A.Q page.

alexevic committed:
Full page TA for F.A.Q page...

...is completed. Also, I made some changes in vision feature file (naming) and made some changes in urls and global-data (added some lines for F.A.Q page).

alexevic committed:
Merge branch 'Indigo' into faq-testing

Deimokas commented on src/features/faq.feature:
```diff
@@ -0,0 +1,176 @@
+
+    Scenario: Check FAQ page title and description
+    Given the user is on the Faq page
+    Then the Faq title, header and description should be correct
```
Could be three steps, one for title, one for header and one for description

alexevic replied:
Test was rewritten, more steps added.

Deimokas commented on src/page-objects/faq.page.ts:
```diff
@@ -0,0 +1,33 @@
+            icon: await getNthElement('div.brighter-accordion-item__toggle.ng-star-inserted > brighter-accordion-item-toggle > brighter-icon > svg > use', nth),
+            title: await getNthElement('[ta-id="faq_201393979"]', nth),
+            description: await getNthElement('[ta-id="faq_166895400"]', nth),
+            item: await getNthElement('[ta-id="faq_217190828"]', nth),
```
Refactor that only relevant methods would be used meaning, move out unused methods

alexevic replied:
Function added to select only one element at the time. Now for test, code gets only elements that are used.

alexevic committed:
Refactored F.A.Q. page TA.

Changed some steps. Added interface for title, header and description F.A.Q. text sections.

alexevic committed:
Merge branch 'Indigo' into faq-testing
